video/out/gpu/video.c

```diff
@@ -3251,9 +3253,13 @@ void gl_video_render_frame(struct gl_video *p, struct vo_frame *frame,
 
     bool has_frame = !!frame->current;
 
-    struct m_color c = p->clear_color;
-    float clear_color[4] = {c.r / 255.0, c.g / 255.0, c.b / 255.0, c.a / 255.0};
-    p->ra->fns->clear(p->ra, fbo.tex, clear_color, &target_rc);
+    if (p->opts.clear_framebuffer == 1 || p->opts.clear_framebuffer == 0 &&
```

Comment:
Needs ( ).

Reply:
just a question on the side, is that some code convention we have and i missed? since technically it works as intended.

Reply:
When mixing && and || (and with some other operators). Not needed when it's a serious of ||. It's not explicitly in the coding style, but most code does it.

Reply:
okay. i think explicitly using brackets here makes it easier to read, since it's explicit. at some point i added them but removed them for the commit again, since i wasn't sure what would be preferred.